src/shared/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { NavLink } from 'react-router-dom';
+
+import IconAdd from '@components/svg/IconAdd';
+import IconHome from '@components/svg/IconHome';
+import IconUsers from '@components/svg/IconUsers';
+
+import './Navbar.scss';
+
+function Navbar() {
+  return (
+    <>
```

Comment:
Tu peux créer une classe tout autour :

```suggestion
    <header className="NavBar">
```